### ISSUE TITLE
create top.sls for a default host-to-formula map

### ISSUE THREAD

**Issue by ketzacoatl** [fpco/fpco-salt-formula#2]
Salt uses a `top.sls` file for _mapping_ a node _target_ to a list of formula. The `top.sls` is a templated YAML file. The _node target_ is a reference to a minion id as a string, possibly with regex and globs (so as to be either exclusive or inclusive). This same method is also used in the pillar subsystem.

Right now, the HPC and Stackage deployments use completely different maps, though only the `hpc-top.sls` file exists (`stackage-top.sls` will come shortly). Thus, there needs to be in place a default `top.sls` that will be used _for all nodes_. This is there if the repo is used _as-is_ (eg an external actor has not overwritten the default `top.sls` with their own) this default list of formula would be applied. This is exactly how packer would run a _base build_ (copy over the formula and run `state.highstate`, don't edit `top.sls`).

In other words this is the current definition of what is in the base image created by Packer (source AMI + default formula). The default formula includes the minimum needed to install and use salt's configuration management and remote execution sub-systems. With the base formula applied, the host would be ready to update itself in the future (based on its state formula and pillar configuration maps), and it would be ready to join a legion under a salt master as a new minion.
